packages/gatsby/src/schema/graphql-engine/bundle-webpack.ts

```diff
@@ -71,17 +78,35 @@ export async function createGraphqlEngineBundle(
     module: {
       rules: [
         {
-          test: /node_modules[/\\]lmdb[/\\].*\.[cm]?js/,
-          parser: { amd: false },
-          use: [
+          oneOf: [
             {
-              loader: require.resolve(`@vercel/webpack-asset-relocator-loader`),
-              options: {
-                outputAssetBase: `assets`,
-              },
+              test: /node_modules[/\\]lmdb[/\\].*\.[cm]?js/,
+              parser: { amd: false },
+              use: [
+                assetRelocatorUseEntry,
+                {
+                  loader: require.resolve(`./lmdb-bundling-patch`),
+                },
+              ],
             },
             {
-              loader: require.resolve(`./lmdb-bundling-patch`),
+              // For node binary relocations, include ".node" files as well here
```

Comment:
Nit: This comment is probably copy/paste and not applicable here?

Reply:
no worries 🤗

Reply:
Updated comments in https://github.com/gatsbyjs/gatsby/pull/35749/commits/d57a2d6ebdfc860c5b994bad8c8f04bf6777377e